web/src/test/java/org/openmrs/web/filter/startuperror/StartupErrorFilterTest.java

```diff
@@ -55,20 +59,18 @@ public void getModel_shouldReturnAStartupErrorFilterModelContainingTheStartupErr
 	@Test
 	public void skipFilter_shouldReturnTrueIfNoErrorHasOccuredOnStartup() {
 		
-		when(Listener.errorOccurredAtStartup()).thenReturn(false);
+		Whitebox.setInternalState(Listener.class, "errorAtStartup", false);
```

Comment:
remove this line, you are setting the errors to null in the after each , so there are no errors by default for this test. the test will then pass.